www/grpc/proto/utils.proto

```diff
@@ -7,11 +7,17 @@ option java_package = "pactus.utils";
 // Utils service defines RPC methods for utility functions such as message
 // signing and verification.
 service Utils {
-  // SignMessageWithPrivateKey sign message with provided private key.
+  // SignMessageWithPrivateKey signs message with provided private key.
   rpc SignMessageWithPrivateKey(SignMessageWithPrivateKeyRequest) returns (SignMessageWithPrivateKeyResponse);
 
-  // VerifyMessage verify signature with public key and message
+  // VerifyMessage verifies signature with public key and message.
   rpc VerifyMessage(VerifyMessageRequest) returns (VerifyMessageResponse);
+
+  // BLSPublicKeyAggregate aggregates bls public keys.
+  rpc BLSPublicKeyAggregate(BLSPublicKeyAggregateRequest) returns (BLSPublicKeyAggregateResponse);
+
+  // BLSSignatureAggregate aggregates bls signatures.
+  rpc BLSSignatureAggregate(BLSSignatureAggregateRequest) returns (BLSSignatureAggregateResponse);
```

Comment:
I think `Aggregation` is better than `Aggregate`

```suggestion
  rpc BLSSignatureAggregation(BLSSignatureAggregationRequest) returns (BLSSignatureAggregationResponse);
```

Reply:
I agree, I will make a commit for it.